Modules/CodeTrek/Entities/CodeTrekApplicant.php

```diff
@@ -22,6 +23,16 @@ public function center()
         return $this->belongsTo(OfficeLocation::class, 'center_id');
     }
 
+    public function getDaysInCodetrekAttribute(self $applicant)
```

Comment:
No need to pass applicant, you can simply use `$this` attribute to get the value of applicant class object
```suggestion
    public function getDaysInCodetrekAttribute()
```